resources/_gen/assets/js/js/search.js_b4bb3e7b21b43e920b7c367ff787d383.content

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2018 Google LLC
```

Comment:
Should this be OpenCue Project Authors?

Reply:
Thanks and I think so. I believe this is coming from the LICENSE file, so I've update that and will regenerate the resources after this is merged.